go.mod

```diff
@@ -1,6 +1,6 @@
 module github.com/bitrise-io/bitrise-init
 
-go 1.22.0
+go 1.23.0
```

Comment:
Could you downgrade this to 1.22.0 as that is the default Go version on some of our stacks. If someone wants to run this on an older stack then it will not compile out of the box.

Reply:
go mod tidy rewrotes it